crates/bevy_render/src/camera/camera.rs

```diff
@@ -415,16 +451,16 @@ impl Camera {
     /// To get the world space coordinates with Normalized Device Coordinates, you should use
     /// [`ndc_to_world`](Self::ndc_to_world).
     ///
-    /// Returns `None` if any of these conditions occur:
-    /// - The logical viewport size cannot be computed. See [`logical_viewport_size`](Camera::logical_viewport_size)
-    /// - The near or far plane cannot be computed. This can happen if the `camera_transform`, the `world_position`, or the projection matrix defined by [`CameraProjection`] contain `NAN`.
-    ///     Panics if the projection matrix is null and `glam_assert` is enabled.
+    /// Panics if the camera's projection matrix is null and `glam_assert` is enabled
+    /// (see [`ndc_to_world`](Self::ndc_to_world).
```

Comment:
I'm not sure what "null" means here. Should this say "contains NaN" instead, or is null referring to something else?

Reply:
contains Nan or is invalid should be fine. Official `glam` docs say:
> Will panic if the determinant of self is zero[..]